qiskit_machine_learning/utils/loss_functions/loss_functions.py

```diff
@@ -196,3 +250,44 @@ def gradient(self, predict: np.ndarray, target: np.ndarray) -> np.ndarray:
         return target * (1.0 / (1.0 + np.exp(-predict)) - 1) + (1 - target) * (
             1.0 / (1.0 + np.exp(-predict))
         )
+
+
+class SVCAlignment(KernelLoss):
+    """
+    This class computes the weighted kernel alignment loss using SKLearn SVC class.
+    """
+
+    def evaluate(
+        self,
+        user_parameters: Sequence[float],
+        kernel: QuantumKernel,
+        data: np.ndarray,
+        labels: np.ndarray,
+    ) -> float:
+        print(f"user parameters: {user_parameters}")
+        # Bind training parameters
+        kernel.assign_user_parameters(user_parameters)
+
+        # Train a quantum support vector classifier
+        kernel_matrix = kernel.evaluate(data, labels)
+        print(f"kernel:\n{kernel_matrix}")
+        svc = SVC(
+            kernel=kernel.evaluate, *self.loss_args, **self.loss_kwargs
```

Comment:
In the latest sklearn release, args went from being deprecated to being completly removed such that SVC only supports named parameters.